.buildkite/bk.integration.pipeline.yml

```diff
@@ -108,7 +122,7 @@ steps:
       - integration-ess
     steps:
       - label: "x86_64:non-sudo: {{matrix}}"
-        # only packaging-ubuntu-x86-64 artifact dependency is required
+        depends_on: packaging-ubuntu-x86-64
```

Comment:
nit: let's stick with one style (can be done in a follow up) either like this, or [like below](https://github.com/elastic/elastic-agent/pull/6972/files#diff-f309b0b791582d35ad84f2c1ef44d26c489f926497a44a3f8e6d72bf3f680572R144) using a list

Reply:
This is about to be removed https://github.com/elastic/elastic-agent/pull/7184